src/app/modules/inbox/inbox.component.html

```diff
@@ -0,0 +1,133 @@
+                            </div>
+                            <a>Php</a>
+                        </md-list-item>
+                        <md-list-item class="subenav">
+                            <div class="logored">
+                                <span>H</span>
+                            </div>
+                            <a>Hybrid Mobile Apps</a>
+                        </md-list-item>
+                        <md-list-item class="subenav">
+                            <div class="logoblue">
+                                <span>T</span>
+                            </div>
+                            <a>testing</a>
+                        </md-list-item>
+                    </md-list>
+                    <md-list-item class="mainnav">
+                        <md-icon md-list-icon class="candcolor">drafts</md-icon>
+                        <a md-line>Candidates</a>
+                    </md-list-item>
+                </md-list>
+
+            </md-sidenav>
+            <div class="content">
+                <div>
+                    <ul class="pagination pull-right">
+                        <li>
+                            <span>
+                           Page no: 1
+                      </span>
+                        </li>
+                        <li class="disabled">
+                            <span>«</span>
+                        </li>
+                        <li><span>»</span>
+                        </li>
+                    </ul>
+                </div>
+                <md-spinner *ngIf="loading" class="posload"></md-spinner>
+                <div *ngIf="!loading">
+                <md-card class="bleftblue emailstyle" *ngFor="let email of emaillist">
+                    <md-grid-list cols="8" rowHeight="100px">
+                        <md-grid-tile [colspan]="'1'">
+                            <div class="emailcheckbox">
+                                <span>S</span>
+                            </div>
+                        </md-grid-tile>
+                        <md-grid-tile [colspan]="'6'">
+                            <div class="detailloc">
+                                <h4 class="inline">{{email.subject}}</h4>
+                                <h6 class="inline">{{email.email_date}}</h6>
+                                <p>{{email.sender_mail}} - From {{email.from}}</p>
+                            </div>
+                        </md-grid-tile>
+                        <md-grid-tile [colspan]="'1'">
+                            <button md-icon-button [mdMenuTriggerFor]="themeMenu">
+                       <md-icon>more_vert</md-icon>
+                      </button>
+                            <md-menu #themeMenu x-position="before">
+                                <button md-menu-item>
+                             <div class="logoorange">
+                                <span>A</span>
+                            </div>
+                           <span>Angular JS</span>
```

Comment:
sir these are tag which we assign to our email. not a menu.  so can not create separate component for these.